pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,108 @@ title: What is JSON Schema?
 
-* Describe your existing data formats
-* Provide clear documentation that’s both human and machine readable
-* Validate your data, which helps you:
-    * Automate testing
-    * Ensure the quality of client-submitted data
+* However, the *modern era*, starting in 2019 and continuing to the present, has brought significant advancements to JSON Schema. They have expanded its capabilities and introduced standardized features. 
 
-JSON Schema is highly extensible and can be tailored to fit your needs. You can create custom keywords, formats, and validation rules to suit your own requirements. The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
+As of now, the current version of JSON Schema is [2020-12](../draft/2020-12/release-notes.md), which represents the latest advancements and refinements in this specification.
 
 
 ## How it works
 
-Using JSON Schema, you can define rules and constraints that JSON data should adhere to. When your JSON documents adhere to these constraints, it becomes easier to exchange structured data between applications because the data follows a consistent pattern.
+1. **Define the rules**: You begin by defining the rules in a JSON Schema document using various [keywords](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-validation-keywords-for-any).
 
-You begin by defining the rules in a JSON Schema document using various [keywords](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-validation-keywords-for-any). For example, you can use `type` to specify what types of data are valid for a given key. You can specify either a single type, like `string`, or an array of possible types, like `string`, `number`, or `integer`. You can also enforce formats using regular expressions with the `pattern` keyword. There are countless ways to describe specific data types and formats that your data must follow.
+    * You can use `type` to specify what types of data are valid for a given key. You can specify either a single type, like `string`, or an array of possible types, like `string`, `number`, or `integer`.
+    * You can also enforce formats using regular expressions with the `pattern` keyword.
+    * There are countless ways to describe specific data types and formats that your data must follow.
 
-After you define your schema, you validate your JSON documents against the schema. To do this, you can use any supported [validator](https://json-schema.org/implementations.html#validators) in whatever language or context fits your project. There are JSON Schema validation tools for nearly every modern programming language, as well as language-agnostic command-line and browser tools.
+2. **Create the schema**: Use the defined rules to create a JSON Schema that represents the expected structure and format of your JSON data.
```

Comment:
IMHO step 1 is in reality part of the step 2.... Can why try to keep both as one single step?